From 7cde409e8402d8890382e9105d6fe4830f9fb7e4 Mon Sep 17 00:00:00 2001
From: Yue Tao <Yue.Tao@windriver.com>
Date: Tue, 21 May 2019 23:36:09 -0700
Subject: [PATCH] mm: miss the #ifdef CONFIG_SMP

Missing the #ifdef CONFIG_SMP while merging upstream commit
6ea183d60c46 mm: handle lru_add_drain_all for UP properly to rt branch.

Signed-off-by: Yue Tao <Yue.Tao@windriver.com>
---
 mm/swap.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mm/swap.c b/mm/swap.c
index 7e0bcaf450a5..e1f4b921c472 100644
--- a/mm/swap.c
+++ b/mm/swap.c
@@ -661,6 +661,8 @@ void lru_add_drain(void)
 	local_unlock_cpu(swapvec_lock);
 }
 
+#ifdef CONFIG_SMP
+
 #ifdef CONFIG_PREEMPT_RT_BASE
 static inline void remote_lru_add_drain(int cpu, struct cpumask *has_work)
 {
-- 
2.17.1

